### DISCUSSION TITLE
Help with ZKG installed packages - I might be dumb.

### DISCUSSION THREAD

**Discussion by scarl3t0** [zeek/zeek#3065] [Help]
Hi all!

Full disclosure this is very likely a dumb question but I have learned it's better to ask and sound dumb than spend a whole day beating my head against a wall. xD

I am running Zeek LTS as a Docker instance. It's completely vanilla apart from the required packages to use zkg per the docs:

_apt-get update
apt-get install -y --no-install-recommends g++ cmake make libpcap-dev_

I also made sure the @load packages function was not commented out.

I issued the following commands in order:
_zeekctl stop
zkg autoconfig
zkg refresh
zkg install icsnpp-enip (this was successful)
zeekctl deploy
zeek -Cr *mypcap*_ 

No enip.log file was created. I then ran:
_zeek -Cr *mypcap* icsnpp/enip_ 

In this case the enip.log file was created just as expected. Can anyone suggest why my zkg installed packages are not loading by default? I'm sure the answer is simple, I have just failed to find the fix so far. Thanks so much.

**Comment by scarl3t0** [comment]
Adding: I tried the same with another icsnpp-s7comm package and had the same result.

**Comment by ckreibich** [comment]
Hi there,

When you run `zeek -Cr mypcap`, your `local.zeek` isn't automatically included. If you say `zeek -Cr mypcap local`, I bet it works, because that in turn will `@load packages`, which will have the same effect as adding `icsnpp/enip` in your last line.

You only need to install the build environment if you want to install `zkg` packages that include plugins (which require compilation to native code), btw, not to use `zkg` in general.

Also, when using `zkg` bundled with Zeek (as in your case), there's no need to run `zkg autoconfig` since it comes pre-configured.

We don't normally provide support in these tickets, btw. Please post questions such as this one on Slack or our Discourse server.

Thanks!
Christian
